go/scripts/shipit

```diff
@@ -51,15 +51,17 @@ godep restore
 go get github.com/mitchellh/gox
 popd
 
+if [[ -n ${STATIC_BINARY} && ${STATIC_BINARY} != "false" ]]; then
+	export CGO_ENABLED=0
+fi
 
 pushd $REPO_ROOT
 mkdir artifacts
-gox -osarch="${TARGETS}" --output="artifacts/${BINARY}-{{.OS}}-{{.Arch}}" -ldflags="-X main.Version=${VERSION}" .
-go build -ldflags="-X main.Version=${VERSION}" .
+gox -osarch="${TARGETS}" --output="artifacts/${BINARY}-{{.OS}}-{{.Arch}}" -ldflags="-X main.Version=${VERSION}" ./...
```

Comment:
Is the `./...` going to work for multi-binary repos like SHIELD, or is gox going to complain about many binaries and one output?